### ISSUE TITLE
Folders are Duplicated

### ISSUE THREAD

**Issue by DTYlan** [kayler-renslow/arma-dialog-creator#22]
### Version and Build
Version: 1.0.2
Build: 15

### Summary
If a folder is moved into another folder and then saved. When reopening the project the folders are duplicated.

### Detail
Imagine the folder structure:
Original:
```
Folder
└── Subfolder
	 ├── Item1
	 └── Item2
```

After Saved and Reopened:
```
Folder
└── Subfolder
	 └── Subfolder
		 ├── Item1
		 └── Item2
```

**Comment by kayler-renslow** [comment]
fixed in https://github.com/kayler-renslow/arma-dialog-creator/commit/d3a08b19659af4357ac23da477eb563ff6b8165b

**Comment by kayler-renslow** [comment]
On a side note, this was related to https://github.com/kayler-renslow/arma-dialog-creator/issues/21

**Comment by kayler-renslow** [comment]
fix applied in version 1.0.3
